### PR TITLE
A few useful recipes

### PR DESCRIPTION
**Pull request by borkurdotnet**
### Description
Added a few field recipes that are useful when creating models. The fields calculate:
- The fraction of population that travels to work using active transport (using census)
- The fraction of green-space in and area (using Open Street Map)
- The area of a geometry in square meters
- The area of a geometry in square kilometers

### Checklist
- [ ] Created new test/s (if possible)
- [ ] Updated the README / documentation, if necessary

### CODE REVIEW

File: src/main/resources/modelling-fields/census/fraction-active-transport-to-work-data.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "importerClass": "uk.org.tombolo.importer.ons.CensusImporter",
+    "datasourceId": "qs701ew"
+  }
+]
```
Comment (eddiejaoude) [inline src/main/resources/modelling-fields/census/fraction-active-transport-to-work-data.json]: GitHub highlights files that dont end with an empty line

Comment (borkurdotnet) [reply src/main/resources/modelling-fields/census/fraction-active-transport-to-work-data.json]: ok. I will add those here.

File: src/main/resources/modelling-fields/census/fraction-active-transport-to-work-field.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (sassalley) [inline src/main/resources/modelling-fields/census/fraction-active-transport-to-work-field.json]: Is this a duplicate of the field outlined [here](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/blob/master/src/main/resources/modelling-fields/city-indices/active-transport/ActiveTransportToWork-field.json)? Potentially under a different modelling field group?

Comment (borkurdotnet) [reply src/main/resources/modelling-fields/census/fraction-active-transport-to-work-field.json]: Yes, this is a duplicate. My idea was to move the re-usable sub-components of the indices into a thematic structure outside the city-indices directory. I.e. that all transport related things are together, all environment together, etc. If this is accepted, the active transport index could use the new component.

That said, going forward, I don't think that directory structure is sufficiently good organisation of model components and we should go for a tagging system along the lines described in #311.

**Comment by borkurdotnet** [comment]
Hold off reviewing this for a while since I am going to add some README files that explain structure of the model directory.

**Comment by sassalley** [comment]
I was going to ask - just looking at the structure now. In `executions > city-indices > active-transport.json` the recipe accesses modelling fields that are further up the file hierarchy e.g:

```
   {
        "fieldClass": "uk.org.tombolo.field.modelling.SingleValueModellingField",
        "label": "index:active_transport",
        "recipe": "city-indices/active-transport/ActiveTransportIndex"
      },
```

Does the recipe 'go back up' the hierarchy to access these fields?

**Comment by borkurdotnet** [comment]
I've added some basic READMEs.

**Pull request merged**